numpy/lib/_npyio_impl.py

```diff
@@ -523,10 +523,12 @@ def save(file, arr, allow_pickle=True, fix_imports=True):
         compatible between different versions of Python).
         Default: True
     fix_imports : bool, optional
-        Only useful in forcing objects in object arrays on Python 3 to be
-        pickled in a Python 2 compatible way. If `fix_imports` is True, pickle
-        will try to map the new Python 3 names to the old module names used in
-        Python 2, so that the pickle data stream is readable with Python 2.
+        The `fix_imports` flag is deprecated and has no effect.
+        .. deprecated:: 2.0
```

Comment:
I would lean towards just not backporting, doesn't seem much of a deal.  @charris want to make the call?